WordPress/src/main/java/org/wordpress/android/ui/mlp/GutenbergPageLayouts.kt

```diff
@@ -0,0 +1,155 @@
+package org.wordpress.android.ui.mlp
+
+//
+// TODO: THIS SHOULD BE MOVED IN RN BRIDGE?
```

Comment:
I put it in the bridge for the iOS side that way ideally we can find a way to share these static layouts in the future.